util/cron/test-memleaks.bash

```diff
@@ -9,5 +9,5 @@ source $CWD/common-localnode-paratest.bash
 
 export CHPL_NIGHTLY_TEST_CONFIG_NAME="memleaks"
 
-$CWD/nightly -cron -memleaks $(memleaks_log full) -no-futures $(get_nightly_paratest_args)
+$CWD/nightly -cron -memleakslog $(memleaks_log full) -no-futures $(get_nightly_paratest_args)
```

Comment:
I think you'll need to remove the other uses of `$(memleaks_log <name>)` from the examples and ml leaks files.